app/components/lazy-minted/LazyMinted.tsx

```
@@ -0,0 +1,107 @@
+              <th className="border border-slate-600 px-4 py-1">Description</th>
+              <th className="border border-slate-600 px-4 py-1">Supply</th>
+              <th className="border border-slate-600 px-4 py-1">More</th>
+            </tr>
+          </thead>
+          <tbody>
+            {nfts.map((nft, i) => (
+              <tr
+                key={nft.metadata.name + '-' + nft.id}
+                onClick={() => handleViewMore(nft)}
+                className="hover:cursor-pointer hover:bg-slate-100 dark:hover:bg-slate-900"
+              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)